src/main/java/com/avengers/gamera/service/CommentService.java

```diff
@@ -25,14 +27,22 @@ public class CommentService {
     private final CommentMapper commentMapper;
     private final UserService userService;
     private final ArticleRepository articleRepository;
+    private final ArticleMapper articleMapper;
+
+    private final ArticleService articleService;
+
 
     public CommentGetDto createNewComment(CommentPostDto commentPostDto) {
+        Article article = articleService.findById(commentPostDto.getArticleId());
+        article.setCommentNum(article.getCommentList().size()+1);
+
         Comment comment = commentMapper.commentPostDtoToComment(commentPostDto);
         comment.setUser(userService.findUser(commentPostDto.getAuthorId()));
+
         if (commentPostDto.getParentId() != null) {
             comment.setParentComment(find(commentPostDto.getParentId()));
         }
-        comment.setArticle(findArticleById(commentPostDto.getArticleId()));
+        comment.setArticle(article);
```

Comment:
同前面一样 直接写id 就行 不需要查一次 再set 一次
```suggestion
        comment.setArticle(Article.Builder().id(commentPostDto.getArticleId()).build());
```